assets/system/game_modules/kami.mjs

```diff
@@ -30,34 +30,87 @@
  *  POSSIBILITY OF SUCH DAMAGE.
 **/
 
-import from from 'from';
-
 export default new
 class Kami
 {
-	constructor(title = Sphere.Game.name)
+	constructor()
 	{
 		this.enabled = SSj.now() > 0;
-		this.title = title;
 		this.records = [];
+		this.outputBordered = false;
+		this.placeholder = new Record();
```

Comment:
What's the purpose of this placeholder Record?

Reply:
For .start() to return when in miniSphere. Enforce always returning the same type to make it easier for Jit to optimise the function.

Reply:
For .start() to return when in miniSphere. Enforce always returning the same type to make it easier for Jit to optimise the function.